app/uk/gov/hmrc/payetaxcalculatorfrontend/controllers/QuickCalcController.scala

```diff
@@ -59,8 +59,16 @@ class QuickCalcController @Inject()(override val messagesApi: MessagesApi,
     }
   }
 
-  def showSalaryForm(): Action[AnyContent] = ActionWithSessionId.async { implicit request =>
-    Future(Ok(salary(Salary.salaryBaseForm)))
+  def showSalaryForm() = ActionWithSessionId.async { implicit request =>
+    cache.fetchAndGetEntry().map {
+      case Some(aggregate) =>
+        val form = {
+          aggregate.savedSalary.map(s => Salary.salaryBaseForm.fill(s)).getOrElse(Salary.salaryBaseForm)
```

Comment:
ignore it for now. We need business requirement to make it more consistent across the calculator.